source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,355 @@
+| EC2 Instance | https://github.com/ministryofjustice/modernisation-platform-terraform-ec2-instance | Module for an ec2 instance creation. | No, region variable when called. |
+| Github OIDC Provider | https://github.com/ministryofjustice/modernisation-platform-github-oidc-provider | This module allows users to create an OIDC Provider and the associated IAM resources required to make use of the connect provider. | No |
+| ECS Cluster | https://github.com/ministryofjustice/modernisation-platform-terraform-ecs-cluster | This repository provides 2 Terraform Modules: Cluster and Service | No, region variable when called. |
+| Pagerduty | https://github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration | | No, region variable when called. |
+| Terraform Module Template | https://github.com/ministryofjustice/modernisation-platform-terraform-module-template | | No |
+| Incident Response | https://github.com/ministryofjustice/modernisation-platform-incident-response | | No |
+| Trusted Advisor | https://github.com/ministryofjustice/modernisation-platform-terraform-trusted-advisor | | No |
+| Lambda Function | https://github.com/ministryofjustice/modernisation-platform-terraform-lambda-function | | No |
+| | | | |
+
+
+
+### Organisational Service Control Policies
+
+
+Service control policies (SCPs) are a type of organization policy that you can use to manage permissions in the organization. SCPs offer central control over the maximum available permissions for all accounts in your organization. SCPs help you to ensure your accounts stay within your organization’s access control guidelines.
+
+
+Currently in `eu-west-1` we can build what is required, however if this region was to change, we would need to check what is limited in which region.
+
+
+### Contact AWS
+
+
+As previously mentioned, contact AWS using the following steps
+
+
+- Raise a ticket, highest priority.
+- Contact our AWS representative for an update on the situation.
+- Organise updates, if the situation changes, our approach might need to.
+
+
+### Running the terraform
+
+
+Once the modules have been updated, and tagged, as well as the PR merged to change any variable or hardcoded mentions of region in `Modernisation-Platform` we will need to begin the rebuild of the platform, starting with networking. The steps below outline which order to follow.
+
+
+## Platform rebuild - Networking
+
+
+This is a large undertaking, and has not been tested but discussed within the team.
+
+
+### Networking Overview
+
+
+Our [Networking Diagram](../concepts/networking/networking-diagram.html) shows a high level view of how shared
+VPCs are connected.
+
+
+Our [NetworkingApproach](../concepts/networking/networking-approach.html) discusses and
+contains a detailed view of our shared VPCs.
+
+
+Our [Transit Gateway](../how-vpcs-access-the-internet.html) and how VPCs access the internet.
+
+
+
+## Stages
```

Comment:
This is a nice breakdown of stages, but it only focuses on networking (and is missing IAM, security baseline, etc.). I think once the networking is recovered, we then need to add other steps of recovering all of the accounts that sit on top of the platform (like for the single account recovery, but skip the part of account creation). Also, will you be even able to run the networking bit, if the IAM roles and policies are not created/updated first?
What about the modernisation-platform account, resources there will need recovering first, won't they?

Reply:
Core accounts need to be created first, then the modernisation platform account. I imagine some parts are joined (understatement of the century) thats why I advise manually running plans when errors occur to debug this. 

 Core acounts
 Modernisation platform account
 member accounts
 
 The only problem is, the networking will never work currently.